### ISSUE TITLE
Vibration on navigation

### ISSUE THREAD

**Issue by Pwuts** [nextcloud/android#10066]
The vibration lasts about 250ms, so it can't be called haptic feedback. Extremely annoying, distracting, and unnecessary.

### Steps to reproduce
1. Download Android app (on Android 12)
2. Navigate to a different view or page in the app

### Expected behaviour
- Silence

### Actual behaviour
- *buzz*

### Environment data
Android version: 12

Device model: Google Pixel 4a

Stock or customized system: stock

Nextcloud app version: 3.19.1

**Comment by AlvaroBrey** [comment]
Please check #8606 matches what you're saying @Pwuts

**Comment by Pwuts** [comment]
This (partly) matches #8606. When I take my NFC cards out of the phone case it indeed stops doing this. But for me it doesn't buzz sporadically: it vibrates always when navigating to a different view.

**Comment by AlvaroBrey** [comment]
Okay @Pwuts, we'll then close this and use #8606 for tracking this as well. We're still waiting on the FIDO lib's dev.

**Comment by AlvaroBrey** [comment]
Duplicate of #8606
